src/app/components/MediaLoader/index.tsx

```diff
@@ -175,13 +180,26 @@ type Props = {
   blocks: MediaBlock[];
   className?: string;
   embedded?: boolean;
+  forcePlaceholderMode?: Mode;
   uniqueId?: string;
 };
 
-const MediaLoader = ({ blocks, className, embedded, uniqueId }: Props) => {
+const MediaLoader = ({
+  blocks,
+  className,
+  embedded,
+  uniqueId,
+  forcePlaceholderMode,
```

Comment:
I am just trying to come up with a way of keeping this, so that we can use it for the ally swarm / storybook, and not remove it later...

What if we were to rename it to simply `placeholderMode`?